examples/simple-service-poc/simple_service/simple_service.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/local/bin/python
+"""
```

Comment:
Maybe state here clearly that this script will be run on providers?

Reply:
I'll add a README for the whole directory

Reply:
https://github.com/golemfactory/yapapi/pull/303/commits/ae7b8c0e6cd0c40c66d1a3651028b8272b2317fd